src/components/Nav.js

```diff
@@ -17,9 +17,9 @@ function LeftSide({isLoggedIn, user, handleLogIn, handleLogOut}) {
   };
 
   return (
-    <FloatLeftMobileNav>
-      <Link style={{verticalAlign: "middle"}} to="/">
-        <img alt="open sauced" src={logo} />
+    <FloatLeftMobileNav >
+      <Link to="/" style={{verticalAlign: "middle", display: "inline-block"}}>
```

Comment:
@JapneetSingh5 Do we need to have style here?

Reply:
The display attr is required, gets messed up on removing it.
The vertical align style was already there, i didn't add it, i guess its fine keeping it inline here as it is unlikely this will be reused or will have an impact going forward.